src/caliper/controllers/controllers.cpp

```diff
@@ -45,6 +45,14 @@ const char* builtin_option_specs =
     " \"query args\"  : { \"level\": \"runtime\", \"group by\": [ \"cupti.runtimeAPI\" ] }"
     "},"
     "{"
+    " \"name\"        : \"profile.kokkos\","
+    " \"type\"        : \"bool\","
+    " \"description\" : \"Profile Kokkos functions\","
+    " \"category\"    : \"region\","
+    " \"services\"    : [ \"kokkostime\"  ],"
+    " \"query args\"  : { \"level\": \"runtime\", \"group by\": [ \"function\" ] }"
```

Comment:
Can skip the query args here. This group by is only needed if you have some custom attribute; if you're using the regular "function" it's fine. This thing is admittedly a bit weird.

Reply:
If I leave query args as an empty dict, it has a problem. Should I just delete the query args field?

Reply:
You can just delete it.

Reply:
Addressed